### ISSUE TITLE
additionalDataManager empty for query returning multiple listItems

### ISSUE THREAD

**Issue by kozuch** [microsoftgraph/msgraph-sdk-java#332]
Query returning one listItem is populated with additionalDataManager. Calling like this:

```
LinkedList<Option> requestOptions = new LinkedList<Option>();
requestOptions.add(new QueryOption("expand", "fields"));
    	
    	ListItem listItem = graphClient
                .sites(mainSiteId)
                .lists(mainListIdOutbox)
                .items(listItemId)
                .buildRequest(requestOptions)
                .get();
    	
    	FieldValueSet fields = listItem.fields;
   	fields.additionalDataManager().get("dmAnnotation").getAsString();

```
Query has this url:
https://graph.microsoft.com/v1.0/sites/XXX/lists/XXX/items/34?expand=fields

But query that returns multiple listItems has empty additionalDataManager:

```
LinkedList<Option> requestOptions = new LinkedList<Option>();
    		requestOptions.add(new QueryOption("expand", "fields"));
    		
    		IListItemCollectionPage listItemPage =  Graph.graphClient
                    .sites(Graph.mainSiteId)
                    .lists(Graph.mainListIdOutbox)
                    .items()
                    .buildRequest(requestOptions)
                    .get();
    		
    		List<ListItem> listOfItems = listItemPage.getCurrentPage(); 
...
FieldValueSet fields = item.fields;
dmID = fields.additionalDataManager().get("dmID").getAsString();
```

The .get() on additionalDataManager results in java.lang.NullPointerException. Query has this url:
https://graph.microsoft.com/v1.0/sites/XXX/lists/XXX/items?expand=fields

**Comment by kozuch** [comment]
Thinking again - this is probably a "feature" of graph-api and not a bug... can someone confirm?

**Comment by MIchaelMainer** [comment]
Thank you for posting this @kozuch . Can you please provide a sanitized capture of the http request and response so we can determine whether this is a client or service issue?

**Comment by baywet** [comment]
Hi,
Thanks for your feedback and patience on the matter.
To me this sounds like a duplicate of #295 but I need to investigate.

**Comment by baywet** [comment]
yes, as I was suspecting this has been fixed by #296 which should be released soon.
Thanks for the feedback!
Closing.
